package.json

```diff
@@ -12,44 +12,55 @@
     "code_format": "run-s code_format:prettier code_format:svgo",
     "code_format:prettier": "prettier --write --list-different .",
     "code_format:svgo": "svgo -q -f icons/",
-    "clean": "shx rm -rf dist",
+    "clean:shieldlib": "cd shieldlib && shx rm -rf dist",
+    "clean:code": "shx rm -rf dist",
+    "clean": "run-s clean:shieldlib clean:code",
     "presprites": "shx rm -rf dist/sprites",
     "sprites": "node scripts/sprites.js",
     "status_map": "node scripts/status_map.js",
-    "taginfo": "node scripts/taginfo.js",
-    "start": "run-s clean sprites serve",
-    "serve": "node scripts/serve.js",
-    "build:code": "node scripts/build.js",
-    "build": "run-s clean sprites build:code taginfo status_map",
-    "test": "mocha",
+    "taginfo": "exec ts-node scripts/taginfo",
+    "start": "run-s clean build:shieldlib sprites shields serve",
+    "serve": "ts-node scripts/serve",
+    "build:shieldlib": "cd shieldlib && node scripts/build.js && cd ..",
```

Comment:
I'd expect the `cd ..` to be unnecessary as the command would get run in an isolated shell.

Reply:
Fixed in ac9bfe3